### ISSUE TITLE
[LDAP] Loading circle hangs but seems to work

### ISSUE THREAD

**Issue by cdamken** [owncloud/core#10869]
### Expected behavior
Connecting to the AD seems to work, but when entering a LDAP filter string in the “User Filter” Tab, the loading circles appear but I get a result.

### Actual behavior
Connecting to the AD seems to work, but when entering a LDAP filter string in the “User Filter” Tab, the loading circles appear but I get no result.

### Steps to reproduce
See the additional Information
There an adjustment in a database table had to be made for the LDAP query to work.
but strangely it now shows the correct users in the "Users" view, but the LDAP plugin still hangs when I click on the "user filter" tab.
So basically it works at the moment, as long as I don't click on one of the Tabs.

### Server specifications
oC6.0.4
Oracle
LDAP with TLS

### Additional Information
S3-ownCloud\Shared\owncloud\support\github-issues\core\10869

@blizzz

**Comment by cdamken** [comment]
Yes :+1:
